docs/database-connections.md

```diff
@@ -1,15 +0,0 @@
-# Database connections in Data API builder
-
-In order to work properly, Data API builder needs to connect to a target database. To do that, a connection string must be supplied in the [configuration file](./configuration-file.md)
-
-## Connection Resiliency
```

Comment:
I couldn't find this doc in the new location. Is that intentional?

Reply:
nvm I found it here: https://learn.microsoft.com/en-us/azure/data-api-builder/database-connections